### ISSUE TITLE
DC Transformation Service - produce correct output, counts, etc.

### ISSUE THREAD

**Issue by patrickzurek** [eXtensibleCatalog/xcmetadataservicestoolkit#842]
JIRA issue created by: rcook
Originally opened: 2010-11-15 08:32 PM

Issue body:
 (nt)

**Comment by patrickzurek** [comment]
JIRA Coment by user: rcook
JIRA Timestamp: 2011-07-14 07:40 PM

Comment body:

We have not tested DC data completely.   Please devise a test that includes:

Running harvest from Rochester URResearch (ask Ben to point you where), running, performing record counting, tie out and validation, get some data validation from Jennifer.  

Once this is done, we will need to do a similar T0, T1, and T2 test with MST AND Drupal.

Will probably need to break this down into smaller tasks.

**Comment by patrickzurek** [comment]
JIRA Coment by user: rcook
JIRA Timestamp: 2011-10-03 04:59 PM

Comment body:

I think this basically means making sure this service works the same and logs the same info as the other services.

**Comment by patrickzurek** [comment]
JIRA Coment by user: jbrand
JIRA Timestamp: 2011-10-04 03:33 PM

Comment body:

I don't fully understand what needs to be done here, especially the last comment "I think this basically means making sure this service works the same and logs the same info as the other services."  Taking a guess at an estimate.  This assumes T0/T1/T2 test defined, passes.

**Comment by patrickzurek** [comment]
JIRA Coment by user: rcook
JIRA Timestamp: 2011-10-04 03:39 PM

Comment body:

Let's break out the T0/1/2 test at the level as we did with Voyager into a separate task that we can save that later.   For now, I think the service should use the same logging features, write to the same logs, track the same sorts for things (failure to produce output, etc) and we should do this with a full UR Research harvest and process.

**Comment by patrickzurek** [comment]
JIRA Coment by user: jbrand
JIRA Timestamp: 2011-10-06 12:18 PM

Comment body:

no time on this one.

**Comment by patrickzurek** [comment]
JIRA Coment by user: jbrand
JIRA Timestamp: 2011-11-08 05:49 PM

Comment body:

tasks remaining for the future include larger scale server testing, adding more rules and finding a good test set to use for the mock harvest test.
